### PR TITLE
refactor: separate NoCollectorModal from CollectorHistoryPage and use useQueryTags composable

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+const handleConfirm = () => {
+    emit('update:visible', false);
+    emit('confirm');
+};
+
+</script>
+
+<template>
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue:26]: question: Why is the template below here?

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue:1]: question: Does this order of template writing also include the code conventions?

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue:26]: It seems default order for Vue3.
My IDE always make vue file like this.

File: apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue (hunk 2 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue:1]: No, we don't have any :)

**Pull request merged**